R/unit.conversions.R

```diff
@@ -37,148 +37,6 @@ NULL
 #' data(unit.conversions); unit.conversions
 NULL
 
-#' Generate units conversion & parsing data
-#' 
-#' Should only need to be run once on a change in the function contents.
-#' Generates the units data to be saved in data/valid.metadata.units, 
-#' data/freeform.unit.translations, data/unit.conversions, and (all three 
-#' combined) R/sysdata.rda. These are then saved with the package.
-#' 
-#' @importFrom stats setNames
-#' @import dplyr
-#' @keywords data units internal
-generateUnitsData <- function() {
```

Comment:
Actually, i think i am going to use my 'right' to modify private functions at will. It didn't seem right to have private and public functions listed together in `?loadflex-deprecated`, so I've added a second file `?loadflex-deprecated-internal` to track the deprecation or removal of internal functions and objects, and I've put generateUnitsData in there.